kolibri/plugins/learn/assets/src/views/ChannelCard.vue

```diff
@@ -119,7 +122,7 @@
     computed: {
       ...mapGetters(['isLearner', 'isUserLoggedIn']),
       isTopic() {
-        return this.kind === ContentNodeKinds.TOPIC || this.kind === ContentNodeKinds.CHANNEL;
```

Comment:
Like I said, I don't think it's necessary to put an `is_topic` field on a serialized Channel object. I'm not sure, but this `isTopic` field might be unnecessary, if the `channels` array in the `contents` prop is all channels. If this is the case, we can just say that `isTopic` is a constant function returning `true`.

https://github.com/learningequality/kolibri/blob/5815bd7d04f368e3e967341f518e4b5c1ee55357/kolibri/plugins/learn/assets/src/views/ChannelsPage.vue#L8-L14